adapters/juniper_srx/prov_init_conn.php

```diff
@@ -10,6 +12,12 @@ function prov_init_conn($sms_csp, $sdid, $sms_sd_info, &$err)
   global $passwd;
   global $port;
 
+
+//  if (isset($sd->SD_CONFIGVAR_list['CUSTOM_MNGT_IP'])) {
+//    $ipaddr = trim($sd->SD_CONFIGVAR_list['CUSTOM_MNGT_IP']->VAR_VALUE);
+//    echo "CUSTOM_MNGT_IP: using custome management port: " . $ipaddr . "\n";
+//  } 
+
```

Comment:
Is it necessary to keep these comments ?

Reply:
@UBiqube-ydu clean up done